### PR TITLE
Add jmespath@0.15.0 w/ npm auto-update

### LINKED ISSUE #11209: [Request] Add jmespath
**Library name:** jmespath
**Git repository url:** https://github.com/jmespath/jmespath.js
**npm package name or url** (if there is one): https://www.npmjs.com/package/jmespath
**License (List them all if it's multiple):** Apache License, Version 2.0
**Official homepage:** http://jmespath.org/
**Wanna say something? Leave message here:** 

=====================

Notes from cdnjs maintainer:
Please read the [README.md](https://github.com/cdnjs/cdnjs#cdnjs-library-repository) and [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md) document first.

We encourage you to add a library via sending pull request,
it'll be faster than just opening a request issue,
since there are tons of issues, please wait with patience,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dakshshah96**
close #11209, cc @jamesls

Pull request for issue: #11209 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/jmespath/jmespath.js
 * Official website (optional, not the repository): http://jmespath.org/
 * NPM package url (optional): https://www.npmjs.com/package/jmespath
 * License and its reference: Apache-2.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 13
   - Count of stars: 117
   - Count of forks: 22
 * NPM download stats (optional):
   - Downloads in the last day: 75,165
   - Downloads in the last week: 443,237
   - Downloads in the last month: 1,918,841

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jmespath/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "jmespath",
+  "version": "0.15.0",
+  "filename": "jmespath.min.js",
+  "description": "JMESPath implementation in JavaScript",
+  "homepage": "http://jmespath.org/",
+  "keywords": [
+    "jmespath",
+    "jsonpath",
```
Comment (extend1994) [inline ajax/libs/jmespath/package.json]: This keyword will direct users to touch another library called JsonPath. I think it's better to remove this.

File: ajax/libs/jmespath/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "json",
+    "xpath"
```
Comment (extend1994) [inline ajax/libs/jmespath/package.json]: This is related XML but this library is a query language for JSON. It might be not suitable.

**Comment by dakshshah96** [comment]
@sufuf3 I tried running the `web-minify-helper` script but it didn't generate a map file.

**Comment by sufuf3** [comment]
@dakshshah96 I remove the `jmespath.min.js` and rerun the `web-minify-helper` script and it can generate a map file.
Would you wanna to try again?
And before run the `web-minify-helper` script. You could `$cd web-minify-helper` and `$git pull origin master` and `$npm update`, let it in latest.

**Comment by PeterDaveHello** [comment]
@sufuf3 npm install before npm update

**Comment by sufuf3** [comment]
Oh, sorry. I'll update the comment.

**Comment by dakshshah96** [comment]
@sufuf3 Done 👍

**Comment by dakshshah96** [comment]
@qwe19272375 Please add `in progress` label here.

**Comment by PeterDaveHello** [comment]
@extend1994 as you are the second reviewer, and there is no rejection/change request here, please request review from @cdnjs/library-reviewer after you reviewed it.

**Pull request merged**